x-pack/plugins/features/common/feature_kibana_privileges.ts

```diff
@@ -216,6 +216,8 @@ export interface FeatureKibanaPrivileges {
      * ```
      */
     settings?: readonly string[];
+    createComment?: readonly string[];
+    reopenCases?: readonly string[];
```

Comment:
Nit: missing doc comments like other interface members

Reply:
Will update, although I think the grammar for all of these doc blocks is a little unclear. Proposed change for all of them: "List of case owners whose users should have settings access when granted this privilege.", "List of case owners whose users should have comment creation access when granted this privilege." etc. cc @cnasikas